link-parent/link-ilp-over-http/src/main/java/org/interledger/link/http/IlpOverHttpLink.java

```diff
@@ -132,12 +132,11 @@ public InterledgerResponsePacket sendPacket(final InterledgerPreparePacket prepa
         Optional<ThrowableProblem> problem = parseThrowableProblem(preparePacket, errorResponseBody);
         final InterledgerRejectPacket rejectPacket;
 
-        if (response.code() == 401 || response.code() == 403) {
-          // If this code is returned, we know the Link is misconfigured, so throw a LinkException Exception.
-          throw new LinkException(String.format(
-              "Unable to connect to remote ILP-over-HTTP Link: Invalid Bearer Token. response=%s", response
-          ), this.getLinkId());
-        } else if (response.code() >= 400 && response.code() < 500) {
+        if (response.code() >= 400 && response.code() < 500) {
+          if (response.code() == 401 || response.code() == 403) {
+            // If this code is returned, we know the Link is misconfigured
+            logger.error("Unable to connect to remote ILP-over-HTTP Link: Invalid Bearer Token. response={}", response);
```

Comment:
I think it would be helpful to put this exception message into the packet's `message` field so that the packet-sender (who won't have access to the log-files containing the emission on 138) can easily see what went wrong.